src/cmd/vm.rs

```diff
@@ -350,7 +359,7 @@ fn get_cheeps_image_name(config: &Config, is_container: bool, branch: &str) -> R
     } else {
         config.arc_vm_cheeps_image().context(
             "Config arc_vm_cheeps_image is not set. For internal users, please configure cheeps \
-             image name for ARC-Container",
+             image name for ARC-container",
```

Comment:
How about just replace them "ARC", not "ARC-container' since technically speaking ARCVM is not a container anymore...
c.f. https://chromeos.dev/en/posts/making-android-more-secure-with-arcvm

Reply:
Thank you for letting me know... it's a typo, so I fixed it.